cli.js

```diff
@@ -11,19 +11,54 @@ const cli = meow(`
 
 	Options
 	  --border-color  Color of the box border [black|red|green|yellow|blue|magenta|cyan|white|gray]
+	  --border-style  Style of the border box [single|double|round|single-double|double-single]
+	                  Can also be passed as a string specifying the characters to use. See example below.
 	  --padding       Space between the text and box border
 	  --margin        Space around the box
 
-	Example
+	Examples
 	  $ boxen I ❤ unicorns
 	  ┌────────────┐
 	  │I ❤ unicorns│
 	  └────────────┘
-`);
+
+	  $ boxen --border-style="double-single" …like everyone
+	  ╒══════════════╕
+	  │…like everyone│
+	  ╘══════════════╛
+
+	  $ boxen --border-style '1234-|' ASCII ftw!
+	  1----------2
+	  |ASCII ftw!|
+	  3----------4
+
+`, {
+	string: 'border-style'
+});
 
 const input = cli.input;
 
+function parseBorderStyle(cli) {
+	const borderStyle = cli && cli.flags && cli.flags.borderStyle;
+	const namedStyles = ['single', 'double', 'round', 'single-double', 'double-single'];
```

Comment:
In `boxen` 0.3.1 I've exposed these, so you can just do `Object.keys(boxen._borderStyles)`.